src/PrestaShopBundle/Form/Admin/Improve/International/Translations/TranslationsSettingsFormHandler.php

```diff
@@ -50,23 +50,23 @@ final class TranslationsSettingsFormHandler implements FormHandlerInterface
     /**
      * @var array the list of Form Types
      */
-    protected $formTypes;
+    protected $form;
 
     /**
-     * @param FormBuilderInterface $formBuilder
+     * @param FormFactoryInterface $formFactory
      * @param HookDispatcherInterface $hookDispatcher
-     * @param array $formTypes
+     * @param string $form
      * @param string $hookName
      */
     public function __construct(
-        FormBuilderInterface $formBuilder,
+        FormFactoryInterface $formFactory,
         HookDispatcherInterface $hookDispatcher,
-        array $formTypes,
-        $hookName
+        string $form,
```

Comment:
Maybe you should keep `$formType` ?

Reply:
not needed anymore in this one, we already have an `array $formTypes` in the `FormHandler`